pages/search.js

```diff
@@ -47,36 +47,63 @@ export default function Search({ data: { users, tags, randUsers } }) {
   const router = useRouter();
   const { username, keyword } = router.query;
   const [notFound, setNotFound] = useState();
-
   const [filteredUsers, setFilteredUsers] = useState(randUsers);
   const [inputValue, setInputValue] = useState(username || keyword || "");
+  
   let results = [];
-
+  
```

Comment:
Blank spaces not required, you can setup Prettier to format this correctly on save in VScode

```suggestion

```